packages/terra-list/src/List.jsx

```diff
@@ -13,41 +18,61 @@ const propTypes = {
    */
   children: PropTypes.node,
   /**
-   * Whether or not the child list items should have a border color applied.
+   * Whether or not the list's child items should have a border color applied.
    */
   isDivided: PropTypes.bool,
   /**
    * Function callback for the ref of the ul.
    */
   refCallback: PropTypes.func,
+  /**
+   * Accessibility role of the list, defaults to 'listbox'. If creating a read-only list, pass 'none', and no role will be added.
+   */
+  role: PropTypes.string,
 };
 
 const defaultProps = {
   children: [],
   isDivided: false,
+  role: 'listbox',
 };
 
 const List = ({
   children,
   isDivided,
   refCallback,
+  role,
   ...customProps
 }) => {
   const listClassNames = cx([
     'list',
-    { divided: isDivided },
     customProps.className,
   ]);
 
+  const attrSpread = {};
+  if (role && role.length > 0 && role !== 'none') {
+    attrSpread.role = role;
+  }
+  if (isDivided) {
+    attrSpread['data-list-items-divided'] = isDivided;
+  }
+
   return (
-    <ul {...customProps} className={listClassNames} ref={refCallback}>
+    <ul {...customProps} {...attrSpread} className={listClassNames} ref={refCallback}>
       {children}
     </ul>
   );
 };
 
 List.propTypes = propTypes;
 List.defaultProps = defaultProps;
-List.Item = ListItem;
 
 export default List;
+export {
```

Comment:
What are the thoughts on creating an `index.js` file which exports each component and update the package.json's main file to point to `lib/index.js` instead of `lib/List.js`?

Reply:
I like the idea of renaming this to index.js

Reply:
Will do.

Reply:
Updated.